spec/lib/msf/ui/console/command_dispatcher/auxiliary_spec.rb

```diff
@@ -1,23 +1,566 @@
+          '192.0.2.1:445         - Cleanup for target 192.0.2.1:445',
+          '192.0.2.1:445 - Check failed: The state could not be determined.',
+          '192.0.2.1:445         - Cleanup for target 192.0.2.1:445'
+        ]
+
+        expect(@combined_output).to match_array(expected_output)
+      end
+
+      it 'runs multiple RHOST values' do
+        current_mod.datastore['RHOSTS'] = '192.0.2.1 192.0.2.2'
+        subject.cmd_check
+        expected_output = [
+          '192.0.2.1:445         - Checking for target 192.0.2.1:445 with normalized datastore value 3.5',
+          '192.0.2.1:445         - Cleanup for target 192.0.2.1:445',
+          '192.0.2.1:445 - Check failed: The state could not be determined.',
+          'Checked 1 of 2 hosts (050% complete)',
+          '192.0.2.2:445         - Checking for target 192.0.2.2:445 with normalized datastore value 3.5',
+          '192.0.2.2:445         - Cleanup for target 192.0.2.2:445',
+          '192.0.2.2:445 - Check failed: The state could not be determined.',
+          'Checked 2 of 2 hosts (100% complete)',
+          'Cleanup for target 192.0.2.1 192.0.2.2:445'
+        ]
+        expect(@combined_output).to match_array(expected_output)
+      end
+
+      it 'normalizes the datastore before running' do
+        current_mod.datastore['RHOSTS'] = '192.0.2.1 192.0.2.2'
+        current_mod.datastore.store('FloatValue', '5.0')
+        subject.cmd_check
+        expected_output = [
+          '192.0.2.1:445         - Checking for target 192.0.2.1:445 with normalized datastore value 5.0',
+          '192.0.2.1:445         - Cleanup for target 192.0.2.1:445',
+          '192.0.2.1:445 - Check failed: The state could not be determined.',
+          'Checked 1 of 2 hosts (050% complete)',
+          '192.0.2.2:445         - Checking for target 192.0.2.2:445 with normalized datastore value 5.0',
+          '192.0.2.2:445         - Cleanup for target 192.0.2.2:445',
+          '192.0.2.2:445 - Check failed: The state could not be determined.',
+          'Checked 2 of 2 hosts (100% complete)',
+          'Cleanup for target 192.0.2.1 192.0.2.2:445'
+        ]
+
+        expect(@combined_output).to match_array(expected_output)
+      end
+
+      it 'supports inline options' do
+        pending "cmd_check doesn't support inline methods, only cmd_run"
+        current_mod.datastore.store('FloatValue', '5.0')
+        subject.cmd_check('RHOSTS=192.0.2.5', 'FloatValue=10.0')
+        expected_output = [
+        ]
+
+        expect(@combined_output).to match_array(expected_output)
+      end
+
+      it 'supports multiple RHOST inline options' do
+        pending "cmd_check doesn't support inline methods, only cmd_run"
+        current_mod.datastore.store('FloatValue', '5.0')
+        subject.cmd_check('RHOSTS=192.0.2.5 192.0.2.6', 'FloatValue=10.0')
+        expected_output = [
+        ]
```

Comment:
When a test is `pending` like this shouldn't the expected output still be filled with what's to be expected when the functionality is fixed?